public/apps/account/account-app.tsx

```diff
@@ -36,6 +37,20 @@ function tenantSpecifiedInUrl() {
 }
 
 export async function setupTopNavButton(coreStart: CoreStart, config: ClientConfigType) {
+  const authType = config.auth?.type;
+  let currAuthType = '';
+  if (typeof authType === 'string') {
+    currAuthType = authType;
+  } else if (typeof authType === 'object' && authType.length === 1) {
```

Comment:
Should `authType === 'object'` be replaced with `Array.isArray(authType)` if it validates that `authType` is an array?

Reply:
Addressed.